akka-discovery/src/main/scala/akka/discovery/Discovery.scala

```diff
@@ -63,7 +63,7 @@ final class Discovery(implicit system: ExtendedActorSystem) extends Extension {
       if (config.hasPath(path)) config.getString(path)
       else
         throw new IllegalArgumentException(
-          s"$path must contain field `class` that is a FQN of a `akka.discovery.ServiceDiscovery` implementation")
+          s"$path must point to a FQN of a `akka.discovery.ServiceDiscovery` implementation")
```

Comment:
As explained in #28550, this message is misleading as it gives the impression that you need a nested `class` field. 

As we can see here: https://github.com/akka/akka/pull/28551/files#diff-131a7333ee160e8ff02426ecbac4ef22R81

The already append a `.class` to the configName.